### PR TITLE
Improve help icon

### PR DESCRIPTION
**Pull request by tripu**
@deniak, I've kept changes separated in two commits: the first one is an exact copy of the CSS you had put in `tr-new` (it wasn't identical to the one in this repo), plus that line in `index.html`; the other one are my actual changes.

### CODE REVIEW

File: mockup1/assets/76f6034.css [hunk 1]
```diff
@@ -453,130 +458,162 @@ ul#container > li::before {
...
+li. {background-color: #f4c96f}
+li. {background-color: #5da29b}
+*/
+
+a.trhelp {
+	border: 0 solid #c6d4e0;
+	border-radius: .75rem;
+	padding: .25rem .5rem;
+	color: black;
+	background-color: #ebf0f4;
```
Comment (deniak) [inline mockup1/assets/76f6034.css]: The background color makes the "?" barely visible with the way that icon appears on apple devices. See screenshot below
photo5573597906791933865

Comment (tripu) [reply mockup1/assets/76f6034.css]: @deniak: I see, you are right. Actually, [that glyph seems too dim on several platforms, even against a white background](https://emojipedia.org/white-question-mark-ornament/). I'm inclined to replace it with a simple question mark in bold, **`?`**. What do you say?

Comment (deniak) [reply mockup1/assets/76f6034.css]: yes, that will simplify things!

**Comment by tripu** [comment]
Ready again, @deniak.

**Pull request merged**